docs/_config.yml

```diff
@@ -15,7 +15,7 @@ navigation:
  - text: About
    url: /about/
  - text: React InstantSearch
-   url: /react-instantsearch/
+   url: /../react-instantsearch/
```

Comment:
no, it won't, because then it's `/instantsearch.js/https://community.algolia.com/react-instantsearch`. This issue will go away all together in twoish weeks with the new docs anyway 😄